account/watcher/watcher_test.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	"github.com/btcsuite/btcd/btcec/v2"
 	secp "github.com/decred/dcrd/dcrec/secp256k1/v4"
-	gomock "github.com/golang/mock/gomock"
+	gomock "go.uber.org/mock/gomock"
```

Comment:
Interesting, so this is a fork?

Reply:
Yeah, apparently the original one was archived. Found out while attempting to fix the broken unit test by upgrading the mock.